src/components/PopupWithForm.js

```diff
@@ -24,10 +24,15 @@ export default class PopupWithForm extends Popup {
     this._form.addEventListener("submit", (evt) => {
```

Comment:
```js

    this._submitButton = this._form.querySelector(".modal__button");
```

You've already found the submit button inside `PopupWithForm`, so you can get it in `index.js`, but you should make this class field **public** by removing the `_` (underscrore) like this:

```js
    this.submitButton = this._form.querySelector(".modal__button");

```